README.md

```diff
@@ -3280,12 +3334,17 @@ Please refer to [CASSANDRA.md](https://github.com/getquill/quill/blob/master/CAS
 
 ### Talks
 
-ScalaDays Berlin 2016 - [Scylla, Charybdis, and the mystery of Quill](https://www.youtube.com/watch?v=nqSYccoSeio)
+ - BeeScala 2019 - [Quill + Spark = Better Together](https://www.youtube.com/watch?v=EXISmUXBXu8)
+ - Scale By the Bay 2019 - [Quill + Doobie = Better Together](https://www.youtube.com/watch?v=1WVjkP_G2cA)
+ - ScalaDays Berlin 2016 - [Scylla, Charybdis, and the mystery of Quill](https://www.youtube.com/watch?v=nqSYccoSeio)
 
 ### Blog posts
 
-[quill-spark: A type-safe Scala API for Spark SQL](https://medium.com/@fwbrasil/quill-spark-a-type-safe-scala-api-for-spark-sql-2672e8582b0d)
-Scalac.io blog - [Compile-time Queries with Quill](http://blog.scalac.io/2016/07/21/compile-time-queries-with-quill.html)
+ - Haoyi's Programming Blog - [Working with Databases using Scala and Quill](http://www.lihaoyi.com/post/WorkingwithDatabasesusingScalaandQuill.html)
+ - Medium @ Juliano Alves - [Quill NDBC Postgres: A New Async Module](https://medium.com/@julianoalves/quill-ndbc-postgres-a-new-async-module-f9cf56c7173c)
```

Comment:
Can you add the links to my blog as references, instead of medium?

https://juliano-alves.com/2019/11/18/contributing-to-quill-a-pairing-session/
and
https://juliano-alves.com/2019/11/29/quill-ndbc-postgres-a-new-async-module/